scripts/make_wheel.sh

```diff
@@ -341,19 +355,25 @@ package_wheel() {
     # Install the wheel and do a smoke test
     unset PYTHONPATH
 
-    $PIP_EXECUTABLE uninstall turicreate # make sure any existing build is uninstalled first
-    $PIP_EXECUTABLE install ${WHEEL_PATH}
+    "$PIP_EXECUTABLE" uninstall -y turicreate
+    "$PIP_EXECUTABLE" install --upgrade "${WHEEL_PATH}"
```

Comment:
well, minimal pkg may come up with different requirements, you don't know. and this `--upgrade` is a soft requirement. If the version meets the requirement, it will skip.

Reply:
The minimal wheel does come with different requirements, but those requirements are a subset of the non-minimal/normal wheel. I'm not understanding why you think this is needed.

Reply:
@hoytak is there any specific reason you put this option `--upgrade` here? Toby has concerns. I'm not sure I should remove it.